virt/virt-2-4-release-notes.adoc

```diff
@@ -0,0 +1,200 @@
+
+//CNV-5422 - CNV support for Ceph-based OCS 4.4.
+
+//CNV-5423 - CDI support for namespaces that are restricted by resource quotas.
+* You can now use the Containerized Data Importer (CDI) to import, upload, and clone virtual machine disks into namespaces
+that are subject to CPU and memory resource restrictions. The default compute resource limits are set to `0` but administrators can configure
+the resource limits applied to CDI worker Pods.
+
+//CNV-5424 - virtctl can now use a DataVolume when uploading virtual machine disks.
+* The `virtctl` tool can now use a `DataVolume` when uploading virtual machine disks to the cluster. This helps prevent virtual machines
+from being inadvertently started before an upload has completed.
+
+//CNV-5421 - DataVolumes have been enhanced with conditions that make it easier to understand the state of VM disk import, clone, and upload.
+
+[id="virt-2-4-web-new"]
+=== Web console
+//Placeholder for new content.
+
+//CNV-5275 - Update template designs - virtual machines and virtual machine templates merged under one menu called Virtualization.
+
+//CNV-5272 - Configure scheduling directly from CNV UI.
+* You can now configure the scheduling properties of virtual machines by accessing the *Scheduling and resources requirements* section of the *Virtual Machine Details* page. For example, you can view and manage affinity rules, dedicated resources, and tolerations for tainted nodes. You can also search for nodes with labels that match specific key/value pairs by using the *Node Selector*.
+
+//CNV-5273 - Attach a config map / secret / service account to a VM from the UI.
+* You can now add xref:../virt/virtual_machines/virt-managing-configmaps-secrets-service-accounts.adoc#virt-managing-configmaps-secrets-service-accounts[secrets, ConfigMaps, and service accounts] to a virtual machine on the *Virtual Machine Overview* -> *Environment* page of the {product-title} web console. You can also remove these resources on the same page.
+
+[id="virt-2-4-changes"]
+== Notable technical changes
+//Placeholder for new content.
+
+//CNV-5465-Disconnected installation
+* {VirtProductName} can be installed on disconnected clusters in restricted network environments that do not have Internet connectivity. You can create local mirrors for the {VirtProductName} Operator, and install the Operator from a local catalog image that is accessible to the disconnected cluster. Learn more about xref:../operators/olm-restricted-networks.adoc#olm-understanding-operator-catalog-images_olm-restricted-networks[using Operator Lifecycle Manager on restricted networks].
+
```

Comment:
We have a KB article describing how to install Openshift Virtualization on a restricted network / disconnected cluster.
Consider linking to that article instead of the OLM general doc for disconnected operators.
https://access.redhat.com/articles/4827691

Reply:
@orenc1 @aburdenthehand Link to the KB article was added.